css/styles.css

```diff
@@ -197,13 +197,13 @@ cursor: pointer;
   margin-left:300px;
   text-align: center;
   padding:10% 0;
+  border-bottom: 0.02px solid white;
 }
 .row{
   display: flex;
   flex-direction: row;
-  padding:5%;
+  padding:0 5%;
```

Comment:
@unadine, kindly revert this change to how it was before. I mean you can remove this zero you added here.